pisek/jobs/parts/completeness_check.py

```diff
@@ -52,3 +64,20 @@ def _evaluate(self) -> None:
 
                 if not ok:
                     self._warn(f"{subtask.name} has no dedicated solution")
+
+    def _check_cms_judge(self) -> None:
+        """Checks that cms judge's stdout & stderr contains only one line."""
+        if self._env.config.judge_type == JudgeType.cms:
+            for judge_out in self._get_judge_outs():
+                with open(judge_out.path) as f:
+                    lines = f.read().rstrip().split("\n")
+                if len(lines) > 1:
```

Comment:
Actually, maybe we should check there is exactly one line. The judge job obviously checks this is the case for stdout, but it doesn't even open stderr.

Reply:
Good point.